packages/meteor/deprecate.js

```diff
@@ -1,3 +1,26 @@
+if (Meteor.isServer && Meteor.isDevelopment) {
+  if (typeof __meteor_runtime_config__ === 'object') {
+    var noDeprecation = process.env.METEOR_NO_DEPRECATION || process.noDeprecation;
+    if (noDeprecation === 'true' || noDeprecation === 'false') {
+      noDeprecation = noDeprecation === 'true';
+    }
```

Comment:
Would this still work? 😄 

```suggestion
    noDeprecation = noDeprecation === 'true';
```

Reply:
The idea is to support this variable as either a boolean or a string regex. The `true/false` comparison ensures proper conversion between the boolean string and its true/false value. Otherwise, the string remains unchanged.

With your proposal, any regex string will always evaluate as `false`.